.github/workflows/draft-pdf.yml

```
@@ -0,0 +1,24 @@
+name: Draft PDF
+on: [push]
```

Only if we push to this path?

```suggestion
on: 
  push:
    paths:
      - docs/paper-joss/paper.md
```